package.json

```diff
@@ -10,6 +10,9 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.1",
```

Comment:
What does @emotion/react have that react doesn't have?

Reply:
Hey Angie, I believe it's a requirement for MUI for it's styling from what I could find

It's usually installed by default together: https://mui.com/material-ui/getting-started/installation/#default-installation